### DISCUSSION TITLE
Is there a way to create&use editor Popover component inside a block tool?

### DISCUSSION THREAD

**Discussion by frankchen211** [codex-team/editor.js#2845] [Q&A]
I want to create a block tool and use Popover component(not through the block setting menu), if it is possible to call the building Popover component?

**Comment by neSpecc** [comment]
It will be available soon. We're gonna move the Popover component to the [@editorjs/utils](https://github.com/editor-js/utils) package

**Reply by frankchen211** [reply]
I would like to create a block tool to have autocomplete function just behave like a code editor autocomplete, if this will be possible by using the Popover component in [@editorjs/utils](https://github.com/editor-js/utils) package? When will it be available?

Thanks!
